packages/react/src/table/Table.stories.tsx

```diff
@@ -267,57 +271,154 @@ export const Actions: Story = {
 								<TextLink href="#">Download</TextLink>
 							</Flex>
 						</TableCell>
-					</tr>
+					</TableRow>
 				</TableBody>
 			</Table>
 		</TableWrapper>
 	),
 };
 
+export const Selectable: Story = {
```

Comment:
This is a new story you can preview here https://design-system.agriculture.gov.au/pr-preview/pr-1353/storybook/index.html?path=/story/content-table--selectable